doc/mrcpp_api/introduction.rst

```diff
@@ -74,7 +86,7 @@ which can be written as the lambda function
 
 .. code-block:: cpp
 
-    auto Z = 1.0;                                 // Hydrogen nuclear charge
+    auto Z = 1.0; // Hydrogen nuclear charge
     auto f = [Z] (const mrcpp::Coord<3> &r) -> double {
```

Comment:
We can kill the `-> double`. I tested it in pilot

Reply:
It's still expressive in the context of documentation, though

Reply:
I would keep it even in the actual code. Lambdas are already hard to parse as is...